tox.ini

```diff
@@ -9,7 +9,7 @@ setenv =
 deps =
     -r requirements-dev.txt
 commands =
-    flake8 .
+    flake8 . --exclude=./.*
```

Comment:
Will this ignore all files?

Reply:
All files and directories that start with a . (dot) which removes the virtualenv.